src/web/releases.rs

```diff
@@ -189,13 +177,13 @@ fn get_search_results(
 
     let statement = "
         SELECT
-            crates.name AS name,
-            releases.version AS version,
-            releases.description AS description,
-            releases.target_name AS target_name,
-            releases.release_time AS release_time,
-            releases.rustdoc_status AS rustdoc_status,
-            repositories.stars AS stars,
+            crates.name,
+            releases.version,
+            releases.description,
+            releases.target_name,
+            releases.release_time,
```

Comment:
Why not simply renaming it as `build_time` here directly?

Reply:
Because it's not the build time, it's the release time. I wanted it to be clear that the name is wrong - it's only called `build_time` below because otherwise I'd have to add another `Option`al field to Release.

Reply:
I meant in the request using `AS` (not sure if this is how you understood my comment so just in case :wink: ).

But with your answer, I htink it's better as it's currently done, more explicit.